packages/components/src/NotificationNext/utils/notificationTemplatePropsCreator.ts

```diff
@@ -0,0 +1,15 @@
+import { type ToastProps } from 'react-toastify-next/dist/types';
+
+import { type NotificationTemplateProps } from '../NotificationTemplateNext';
+import { type NotificationProps, type Variant } from '../types';
+
+export const notificationTemplatePropsCreator = (
```

Comment:
https://kaluga-astral.github.io/style-guide/docs/rules/naming/functions#%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D0%B8-%D0%B8-%D0%BC%D0%B5%D1%82%D0%BE%D0%B4%D1%8B-%D0%B4%D0%BE%D0%BB%D0%B6%D0%BD%D1%8B-%D0%BD%D0%B0%D1%87%D0%B8%D0%BD%D0%B0%D1%82%D1%8C%D1%81%D1%8F-%D1%81-%D0%B3%D0%BB%D0%B0%D0%B3%D0%BE%D0%BB%D0%B0